unlock-app/src/__tests__/utils/wedlocks.test.js

```diff
@@ -54,4 +54,11 @@ describe('verifyEmailSignature', () => {
       )
     ).toBe(false)
   })
+
+  it('pulls the public key out of the config file if not provided in arguments', () => {
```

Comment:
The `base64WedlocksPublicKey` variable is optional, but the test didn't previously exercise the branch where it is not provided.